src/main.cpp

```diff
@@ -3,14 +3,89 @@
 //  Game of Life
 //
 
+#include <SFML/Graphics.hpp>
 #include <cmath>
 #include <iostream>
+#include "Grid.hpp"
 
 
 namespace {
 
+const int SCREEN_WIDTH = 1420;
+const int SCREEN_HEIGHT = 1000;
+
 void run(char** argv) {
-    printf("Hello, world!\n");
+    
+    sf::Clock clock;
+    sf::Time elapsed;
+    
+    int time_step = 64;
+    bool paused = false;
+    
+    //some SFML variables for drawing a basic window + background
+    auto window = sf::RenderWindow{sf::VideoMode{SCREEN_WIDTH, SCREEN_HEIGHT}, "Game of Life v1.0"};
+    sf::RectangleShape background = sf::RectangleShape();
```

Comment:
Nitpick: don't need to call the constructor explicitly, just `sf::RectangleShape background{};` is enough.